README.md

```diff
@@ -30,8 +30,8 @@ $ npx sequelize db:seed:all # Run seed file for seeding database
 
 Create a `.env` file in the root directory of the project. Add `.env` to the `.gitignore` file. Make sure to insert the secret key without the alligator clips ( < > ).
 ```
-edamam_id=<edamam_id>
-edamam_key=<edamam_key>
+EDAMAM_ID=<edamam_id>
+EDAMAM_KEY=<edamam_key>
 ```
```

Comment:
Thanks good catch. I don't know if there's a correct "standard" way or if lower case would've worked. My atom just liked it better in terms of coloring and thats how Ive always seen it.

Reply:
I'm not sure what the standard is either, but I've definitely seen it at a higher frequency with all caps.